### PR TITLE
[CI] Enable AOT inductor in cpu performance smoke test

### PR DESCRIPTION
**Pull request by zxd1997066**
cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames

### CODE REVIEW

File: benchmarks/dynamo/expected_ci_speedup_inductor_torchbench_cpu.csv [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-vgg16,amp,static,default,1.267194285
-hf_Longformer,amp,dynamic,default,0.997006035
-hf_Bert_large,amp,dynamic,default,0.99391146
-llama,amp,static,default,1.32950568
-timm_regnet,amp,static,cpp,1.157188305
-lennard_jones,amp,static,cpp,2.240104485
-#hf_T5_generate,amp,dynamic,cpp,1.29339502
-timm_vovnet,amp,dynamic,cpp,1.07856471
-mobilenet_v2,amp,dynamic,cpp,2.27774577
+#name,data_type,shape,wrapper,inductor_type,perf_speedup_target_c7i_metal_24xl
```
Comment (chuanqi129) [inline benchmarks/dynamo/expected_ci_speedup_inductor_torchbench_cpu.csv:1]: let's add the `backend` in the csv title, and please use the full backend name directly. default -> inductor, aot -> export-aot-inductor. And I also suggest to move the backend column at the beginning.

File: .ci/pytorch/test.sh [hunk 2]
```diff
@@ -667,22 +667,29 @@ test_inductor_torchbench_cpu_smoketest_perf(){
...
-    local speedup_target=${model_cfg[4]}
+    local speedup_target=${model_cfg[5]}
+    local inductor_type=${model_cfg[4]}
     if [[ ${model_cfg[3]} == "cpp" ]]; then
       export TORCHINDUCTOR_CPP_WRAPPER=1
     else
       unset TORCHINDUCTOR_CPP_WRAPPER
     fi
+
+    if [[ ${model_cfg[4]} == "aot" ]]; then
```
Comment (chuanqi129) [inline .ci/pytorch/test.sh:678]: If we use full backend name directly, we don't need this part, right?

File: benchmarks/dynamo/expected_ci_speedup_inductor_torchbench_cpu.csv [hunk 3]
```diff
@@ -1,21 +1,26 @@
...
-hf_Bert_large,amp,dynamic,default,0.99391146
-llama,amp,static,default,1.32950568
-timm_regnet,amp,static,cpp,1.157188305
-lennard_jones,amp,static,cpp,2.240104485
-#hf_T5_generate,amp,dynamic,cpp,1.29339502
-timm_vovnet,amp,dynamic,cpp,1.07856471
-mobilenet_v2,amp,dynamic,cpp,2.27774577
+#name,backend,data_type,shape,wrapper,perf_speedup_target_c7i_metal_24xl
+#timm_vision_transformer,backend=inductor,float32,static,default,1.039510755
+phlippe_densenet,backend=inductor,float32,static,default,1.3988316
```
Comment (chuanqi129) [inline benchmarks/dynamo/expected_ci_speedup_inductor_torchbench_cpu.csv:3]: Let's use the `inductor` & `export-aot-inductor` in here directly

**Review by chuanqi129 — APPROVED** [review]

**Review by desertfire — APPROVED** [review]

**Pull request closed without merging**